pilot/pkg/features/pilot.go

```diff
@@ -135,6 +135,12 @@ var (
 		false,
 		"If enabled, controller that untaints nodes with cni pods ready will run. This should be enabled if you disabled ambient init containers.").Get()
 
+	EnableV2IPAutoallocate = env.Register(
+		"PILOT_ENABLE_V2_IP_AUTOALLOCATE",
```

Comment:
nit: I would remove "v2" from anything user facing. I don't care too much about this though since its a feature flag though

Reply:
ty!